### PR TITLE
ee: add ee integration tests

### LINKED ISSUE #357: Add integration tests for EE functionality
Since we have the ability to pass ee-credentials from environment variables, we need to test ee-related functionality in CI.

### PR DESCRIPTION
**Pull request by vr009**
With this patch:
- added integration tests for search/install tarantool-ee
- added a new pytest tag for ee tests
- included ee tests in full-ci target

Closes #357

### CODE REVIEW

File: test/integration/ee/test_ee_install.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    rc, output = run_command_and_get_output(
+        [tt_cmd, "install", "tarantool-ee="+version],
+        cwd=tmpdir, env=dict(os.environ, PWD=tmpdir))
+
+    assert rc == 0
+    assert re.search("Installing tarantool-ee="+version, output)
+    assert re.search("Downloading tarantool-ee...", output)
+    assert re.search("Done.", output)
+    assert os.path.exists(os.path.join(tmpdir, 'bin', 'tarantool'))
```
Comment (psergee) [inline test/integration/ee/test_ee_install.py:36]: Include dir?

**Review by psergee — APPROVED** [review]

File: .github/workflows/full-ci.yml [hunk 2]
```diff
@@ -144,5 +144,120 @@ jobs:
...
+          TT_CLI_BUILD_SSL: 'static'
+        run: mage build
+
+      - name: Integration tests
+        env:
+          TT_EE_USERNAME: '${{ secrets.TT_EE_USERNAME }}'
+          TT_EE_PASSWORD: '${{ secrets.TT_EE_PASSWORD }}'
+        run: mage integrationee
+
+  full-ci-sdk-ee:
```
Comment (psergee) [inline .github/workflows/full-ci.yml:183]: Please add a comment for "sdk-ee", "ce-ee": what do these suffixes mean?

File: test/integration/ee/test_ee_install.py (hunk 1 above)
Comment (vr009) [reply test/integration/ee/test_ee_install.py:36]: Added the check.

File: .github/workflows/full-ci.yml (hunk 2 above)
Comment (vr009) [reply .github/workflows/full-ci.yml:186]: Added.

**Review by psergee — APPROVED** [review]

**Pull request merged**